dockerfiles/theia-endpoint-runtime/Dockerfile

```diff
@@ -48,7 +55,12 @@ COPY /docker-build/theia-plugin /home/workspace/packages/theia-plugin
 COPY /docker-build/theia-plugin-remote/tsconfig.json /home/workspace/packages/theia-plugin/tsconfig.json
 
 COPY /etc/package.json /home/workspace
-RUN cd /home/workspace/ && yarn install
+# installing all dependencies
+RUN cd /home/workspace/ && yarn install --ignore-scripts
+# remove all @theia/* modules, and replace tham from builded Theia branch version
+RUN rm -rf /home/workspace/node_modules/@theia && cp -r -L ${HOME}/theia-source-code/node_modules/@theia /home/workspace/node_modules/
```

Comment:
hum does it mean that we have tons of modules that are not required ?

I'm kind of -1 for this hot replacement.

Reply:
That only for `@theia/*, I not copy all modules.
I'm not very happy with this also, can you suggest better solution?

Reply:
as we're compiling theia, can't we just do like we do for theia ? like copying remote packages to `packages` folder
then create examples/assembly folder and copy the package.json that we have for theia-remote
and then just call che-theia-init:production so that it will create a single folder that we can use as theia-remote with all the right dependencies ?

Reply:
OK, I can try

Reply:
Unfortunately  `che:theia production` doesn't work with any node project, it works only for `@eclipse-che/theia-assembly` https://github.com/eclipse/che-theia/blob/master/generator/src/production.ts#L165
Or we need to change che-theia generator :(

Reply:
is it an issue to name it theia-assembly before running the production command ?

Reply:
No,I update PR